packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -14,150 +14,141 @@ limitations under the License.
 import React from 'react';
 import { injectIntl } from 'react-intl';
 import { Link } from 'react-router-dom';
-import {
-  StructuredListBody,
-  StructuredListCell,
-  StructuredListHead,
-  StructuredListRow,
-  StructuredListWrapper
-} from 'carbon-components-react';
+import { Table } from '@tektoncd/dashboard-components';
+import 'carbon-components-react';
 import { getStatus, getStatusIcon, urls } from '@tektoncd/dashboard-utils';
 
 import { FormattedDate, RunDropdown } from '..';
 
 import './PipelineRuns.scss';
 
 const PipelineRuns = ({
-  createPipelineRunURL = urls.pipelineRuns.byName,
-  createPipelineRunDisplayName = ({ pipelineRunMetadata }) =>
-    pipelineRunMetadata.name,
-  createPipelineRunTimestamp = pipelineRun =>
-    getStatus(pipelineRun).lastTransitionTime,
-  createPipelineRunsByPipelineURL = urls.pipelineRuns.byPipeline,
-  getPipelineRunStatus = (pipelineRun, intl) =>
-    pipelineRun.status && pipelineRun.status.conditions
-      ? pipelineRun.status.conditions[0].message
-      : intl.formatMessage({
-          id: 'dashboard.pipelineRuns.status.pending',
-          defaultMessage: 'Pending'
-        }),
-  getPipelineRunStatusIcon = pipelineRun => {
-    const { reason, status } = getStatus(pipelineRun);
-    return getStatusIcon({ reason, status });
-  },
```

Comment:
These props are used for the integration in IBM Cloud DevOps, and allow 3rd party consumers to override the default dashboard URL structure, etc. We'll need to continue supporting these unless we have an alternative that'll work with minimal impact

Reply:
Sorry I didn't realise, I've updated the code to use these 😸

Reply:
Not a problem. I have a todo item on my list to start adding some structured documentation on our published components to prepare them for broader use. This should help make it clearer what is/isn't considered part of our APIs.